Code/Tools/AssetProcessor/native/ui/ProductAssetDetailsPanel.h

```diff
@@ -10,6 +10,8 @@
 
 #if !defined(Q_MOC_RUN)
 #include "AssetDetailsPanel.h"
+#include "ProductDependencyTreeModel.h"
```

Comment:
the coding standard is to prefer the include <> over include "" when possible
would it be possible to use "include<>" instead?

Reply:
I found that existing files, such as [SourceAssetDetailsPanel.cpp](https://github.com/aws-lumberyard-dev/o3de/blob/sven_ProductAssetDetail/Code/Tools/AssetProcessor/native/ui/SourceAssetDetailsPanel.cpp), ProductAssetTreeModel.cpp ...etc, also includes headers in the same directory in this manner. Could it be actually a proper way? Or perhaps these includes have to be changed as well?

Reply:
changed to angle bracket